packages/environment-glimmerx/__tests__/component.test.ts

```diff
@@ -65,9 +65,18 @@ import { AcceptsBlocks, EmptyObject } from '@glint/template/-private/integration
   }
 
   class YieldingComponent<T> extends Component<YieldingComponentSignature<T>> {
-    static template = template(function* <T>(𝚪: ResolveContext<YieldingComponent<T>>) {
-      expectTypeOf(𝚪.this).toEqualTypeOf<YieldingComponent<T>>();
-      expectTypeOf(𝚪.args).toEqualTypeOf<{ values: T[] }>();
+    static template = templateForBackingValue(this, function (𝚪) {
+      // We can't directly assert on the type of e.g. `@values` here, as we don't
+      // have a name for it in scope. However, the yields below confirm that the
+      // only thing we can legally yield to the default block is an element of the
```

Comment:
```suggestion
        // have a name for it in scope: the type `T` is present on the class instance,
        // but not in a `static` block. However, the yields below confirm that the
```

Reply:
The extra indentation here infuriated CI 😂😭